projects/js-packages/publicize-components/src/components/social-post-modal/post-preview.tsx

```diff
@@ -55,10 +55,9 @@ export function PostPreview( { connection }: PostPreviewProps ) {
 	);
 
 	const siteName = useSelect( select => {
-		// @ts-expect-error `getUnstableBase` exists in the store but is not typed
 		const { getUnstableBase } = select( coreStore );
 
-		return decodeEntities( getUnstableBase()?.name );
+		return decodeEntities( getUnstableBase( undefined )?.name );
```

Comment:
This is a nice workaround for now. We can definitely improve that in core in future.

Reply:
Looks like Gutenberg has swapped to [using `getEntityRecord` for this](https://github.com/WordPress/gutenberg/pull/64884/files#diff-9dd774fe20a86a51fbcd56da7dbca8edc928df53de05a9e9bbee418083b86997R131)